### DISCUSSION TITLE
Center the NavBar

### DISCUSSION THREAD

**Discussion by alpachinois** [havit/Havit.Blazor#838] [General]
Hi,

I try to use your NavBar component but I'd like to center all content. I have a logo on the left and 3 NavLink. All examples put them on the left side.

Thanks for your help.

**Comment by crdo** [comment]
@alpachinois Set `d-flex justify-content-center` to `ContainerCssClass` parameter of `HxNavbar` and `flex-grow-0` on the CssClass parameter of `HxNavbarCollapse`.
